SCons/Tool/docbook/docbook-xsl-1.76.1/webhelp/template/content/search/es-es.props

```diff
@@ -1,179 +1,179 @@
-DEF94=nos
-DEF95=nosotros
-DEF96=vosotros
-DEF97=vosotras
-DEF98=si
-DEF99=dentro
-DEF100=solo
-DEF101=solamente
-DEF102=saber
-DEF103=sabes
-DEF104=sabe
-DEF105=sabemos
-DEF106=sabeis
-DEF107=saben
-DEF108=ultimo
-DEF109=largo
-DEF110=bastante
-DEF111=haces
-DEF112=muchos
-DEF113=aquellos
-DEF114=aquellas
-DEF115=sus
-DEF116=entonces
-DEF117=tiempo
-DEF118=verdad
-DEF119=verdadero
-DEF120=verdadera
-DEF121=cierto
-DEF122=ciertos
-DEF123=cierta
-DEF124=ciertas
-DEF125=intentar
-DEF126=intento
-DEF127=intenta
-DEF128=intentas
-DEF129=intentamos
-DEF130=intentais
-DEF131=intentan
-DEF132=dos
-DEF133=bajo
-DEF134=arriba
-DEF135=encima
-DEF136=usar
-DEF137=uso
-DEF138=usas
-DEF139=usa
-DEF140=usamos
-DEF141=usais
-DEF142=usan
-DEF143=emplear
-DEF144=empleo
-DEF145=empleas
-DEF146=emplean
-DEF147=ampleamos
-DEF148=empleais
-DEF149=valor
-DEF150=muy
-DEF151=era
-DEF152=eras
-DEF153=eramos
-DEF154=eran
-DEF155=modo
-DEF156=bien
-DEF157=cual
-DEF158=cuando
-DEF159=donde
-DEF160=mientras
-DEF161=quien
-DEF162=con
-DEF163=entre
-DEF164=sin
-DEF165=trabajo
-DEF166=trabajar
-DEF167=trabajas
-DEF168=trabaja
-DEF169=trabajamos
-DEF170=trabajais
-DEF171=trabajan
-DEF172=podria
-DEF173=podrias
-DEF174=podriamos
-DEF175=podrian
-DEF176=podriais
-DEF177=yo
-DEF178=aquel
+DEF01=un
+DEF02=una
+DEF03=unas
+DEF04=unos
+DEF05=uno
+DEF06=sobre
+DEF07=todo
+DEF08=tambi�n
+DEF09=tras
+DEF10=otro
+DEF11=alg�n
+DEF12=alguno
+DEF13=alguna
+DEF14=algunos
+DEF15=algunas
+DEF16=ser
+DEF17=es
+DEF18=soy
+DEF19=eres
+DEF20=somos
+DEF21=sois
+DEF22=estoy
+DEF23=esta
+DEF24=estamos
+DEF25=estais
+DEF26=estan
+DEF27=como
+DEF28=en
+DEF29=para
+DEF30=atras
+DEF31=porque
+DEF32=por 
+DEF33=estado
+DEF34=estaba
+DEF35=ante
+DEF36=antes
+DEF37=siendo
+DEF38=ambos
+DEF39=pero
+DEF40=por
+DEF41=poder
+DEF42=puede
+DEF43=puedo
+DEF44=podemos
+DEF45=podeis
+DEF46=pueden
+DEF47=fui
+DEF48=fue
+DEF49=fuimos
+DEF50=fueron
+DEF51=hacer
+DEF52=hago
+DEF53=hace
+DEF54=hacemos
+DEF55=haceis
+DEF56=hacen
+DEF57=cada
+DEF58=fin
+DEF59=incluso
+DEF60=primero
+DEF61=desde
+DEF62=conseguir
+DEF63=consigo
+DEF64=consigue
+DEF65=consigues
+DEF66=conseguimos
+DEF67=consiguen
+DEF68=ir
+DEF69=voy
+DEF70=va
+DEF71=vamos
+DEF72=vais
+DEF73=van
+DEF74=vaya
+DEF75=gueno
+DEF76=ha
+DEF77=tener
+DEF78=tengo
+DEF79=tiene
+DEF80=tenemos
+DEF81=teneis
+DEF82=tienen
+DEF83=el
+DEF84=la
+DEF85=lo
+DEF86=las
+DEF87=los
+DEF88=su
+DEF89=aqui
+DEF90=mio
+DEF91=tuyo
+DEF92=ellos
+DEF93=ellas
+DEF94=nos
+DEF95=nosotros
+DEF96=vosotros
+DEF97=vosotras
+DEF98=si
+DEF99=dentro
+DEF100=solo
+DEF101=solamente
+DEF102=saber
+DEF103=sabes
+DEF104=sabe
+DEF105=sabemos
+DEF106=sabeis
+DEF107=saben
+DEF108=ultimo
+DEF109=largo
+DEF110=bastante
+DEF111=haces
+DEF112=muchos
+DEF113=aquellos
+DEF114=aquellas
+DEF115=sus
+DEF116=entonces
+DEF117=tiempo
+DEF118=verdad
+DEF119=verdadero
+DEF120=verdadera
+DEF121=cierto
+DEF122=ciertos
+DEF123=cierta
+DEF124=ciertas
+DEF125=intentar
+DEF126=intento
+DEF127=intenta
+DEF128=intentas
```

Comment:
> __[Misspell]__ "intentas" is a misspelling of "intents" ([view](https://sider.review/gh/repos/104670160/pulls/4220#issue-7880431))
>
> <details>
>
> | Rule | Correct | Incorrect |
> | ---- | ------- | --------- |
> | `intents` | intents | intentas |
>
> You can [close this issue](https://sider.review/gh/repos/104670160/pulls/4220#issue-7880431) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

This is from the vendored docbook-xsl files, and it's not misspelled, it's in another language. Ignore.